src/js/browser-plugin.js

```diff
@@ -1300,6 +1293,7 @@ function sendLabelEvaluation(targetButton, tweetInfo, agreement) {
 
 function claimClickAction(tweet) {
 
+  log2Server('claim', tweet.url, `Tweet id: ${tweet.id}\nTweet label: ${tweet.coInformLabel}`, `send a claim for a ${tweet.coInformLabel} tweet`);
```

Comment:
Here the claim is not yet sent, even is possible that if the user is not logged in, the claim popup will not even be opened.
This log should be moved inside the "openClaimPopup" function, and the action message should be changed by "open the claim popup ..."